code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -204,6 +204,30 @@
 		if("harm")
 			wuv(-1, M)
 
+/mob/living/simple_animal/pet/cat/attack_alien(mob/living/carbon/alien/humanoid/M)	//what the xeno was after all along in Alien
+	if(M.a_intent == "help" && M.caste == "h")
+		if (icon_state == "cat" || icon_state == "cat_rest" || icon_state == "cat_sit")
+			visible_message("<span class='name'>[M]</span> picks up and hugs <span class='name'>[src]</span>.")
+			src.Move(M)
+
+			M.sprite_changed_for_emote = TRUE
+			var/original_icon = M.icon
+			var/original_icon_state = M.icon_state
+
+			M.icon = 'icons/mob/alien_emotes.dmi'
+			M.icon_state = "cat_cuddle"
+
+			spawn(M.emote_length)
+				M.icon = original_icon
+				M.icon_state = original_icon_state
+				M.sprite_changed_for_emote = FALSE
+
+				src.forceMove(M.loc)
+		else
+			M << "It looks wrong... why would you pick that one up?"

```

Comment:
Oh, okay. I'll try to read the style sheet for what else I can use though because I thought any red font was reserved for you're/something's taking (potential) damage?